webapp/src/components/atoms/UsersQuery.tsx

```diff
@@ -4,6 +4,7 @@ const getUsersQuery = `
   query {
```

Comment:
Can you add `import React from 'react';` to the top of this file? That should fix many of the lint errors